src/components/ExportMenu.jsx

```diff
@@ -88,142 +88,6 @@ export const ExportMenu = () => {
     }
   }
 
```

Comment:
Cleared old "download" code from before, new download code can be accesses in download-utils